test/test_grid_file.py

```diff
@@ -447,27 +462,85 @@ def test_readline(self):
         self.assertEqual(b"e", g.readline(1))
         self.assertEqual(b"llo world,\n", g.readline())
 
+    def test_readlines(self):
```

Comment:
Not sure if this test should be removed; it tests `readlines` which is just the IOBase implementation of `readlines` that internally uses GridOut `readline`, and `readline` is already tested in `test_readline`.

Reply:
Let's keep this test. It proves that readlines() actually works.